R/help.R

```diff
@@ -238,33 +248,63 @@ help_formals_handler.python.builtin.object <- function(topic, source) {
   NULL
 }
 
+sphinx_doc_params_pattern <- function() { "\nParameters\n+[-]+" }
+
+sphinx_doc_params_matches <- function(doc) {
+  regexpr(pattern = sphinx_doc_params_pattern(), doc)
+}
+
+is_sphinx_doc <- function(doc) {
+  sphinx_doc_params_matches(doc)[[1]] != -1
+}
+
+sphinx_doctree_from_doc <- function(doc) {
+  module_name <- "docutils"
+  if (py_module_available(module_name)) {
+    docutils <- import(module_name)
+    docutils$core$publish_doctree(doc)
```

Comment:
Could this ever fail? (documentation cannot be parsed; differences in Sphinx documentation structure between different versions of `docutils`)